sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/application/contacts/EditContactPersonSteps.java

```diff
@@ -0,0 +1,100 @@
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.sormas.e2etests.steps.application.contacts;
+
+import static org.sormas.e2etests.pages.application.contacts.EditContactPersonPage.*;
+
+import cucumber.api.java8.En;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import javax.inject.Inject;
+import org.assertj.core.api.SoftAssertions;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+import org.sormas.e2etests.pojo.Contact;
+import org.sormas.e2etests.steps.web.application.contacts.CreateNewContactSteps;
+
+public class EditContactPersonSteps implements En {
+
+  private final WebDriverHelpers webDriverHelpers;
+  protected Contact aContact;
+
+  @Inject
+  public EditContactPersonSteps(WebDriverHelpers webDriverHelpers, final SoftAssertions softly) {
+    this.webDriverHelpers = webDriverHelpers;
+
+    When(
+        "I check the created data is correctly displayed on Edit Contact Person page",
+        () -> {
+          aContact = collectContactData();
+          softly
+              .assertThat(aContact.getFirstName())
+              .isEqualToIgnoringCase(CreateNewContactSteps.contact.getFirstName());
+          softly
+              .assertThat(aContact.getLastName())
+              .isEqualToIgnoringCase(CreateNewContactSteps.contact.getLastName());
+          softly
+              .assertThat(aContact.getDateOfBirth())
+              .isEqualTo(CreateNewContactSteps.contact.getDateOfBirth());
+          softly.assertThat(aContact.getSex()).isEqualTo(CreateNewContactSteps.contact.getSex());
+          softly
+              .assertThat(aContact.getNationalHealthId())
+              .isEqualTo(CreateNewContactSteps.contact.getNationalHealthId());
+          softly
+              .assertThat(aContact.getPassportNumber())
+              .isEqualTo(CreateNewContactSteps.contact.getPassportNumber());
+          softly
+              .assertThat(aContact.getPrimaryEmailAddress())
+              .isEqualTo(CreateNewContactSteps.contact.getPrimaryEmailAddress());
+          softly
+              .assertThat(aContact.getPrimaryPhoneNumber())
+              .isEqualTo(CreateNewContactSteps.contact.getPrimaryPhoneNumber());
+          softly.assertAll();
+        });
+  }
+
+  public Contact collectContactData() {
+    webDriverHelpers.scrollToElement(CONTACT_PERSON_TAB);
+    webDriverHelpers.clickOnWebElementBySelector(CONTACT_PERSON_TAB);
+    webDriverHelpers.waitUntilIdentifiedElementIsPresent(PERSON_UUID_INPUT);
```

Comment:
you don't need this because on the line 91 `getTextFromWebElement` contains a wait